MainModule/Server/Core/Core.luau

```diff
@@ -1289,9 +1289,11 @@ return function(Vargs, GetEnv)
 								local data = GetData(tData.TableKey)
 								if data then
 									--// TODO: Possibly find a better way to "batch" TableUpdates to prevent script exhaustion
+									local isStudio = service.RunService:IsStudio() -- Studio lag workaround for loading larger data such as BannedList
+									local isHugeTable = #data > 1000
```

Comment:
Perhaps increase this to 3000 at least?

Reply:
The lags in processing 8k+ of banlists lasted around 6-8 seconds (i7-4770 + 16GB + SSD) so I think 1000 is enough for minimizing the startup lag (especially if the developers have weaker computers and larger banlist)